stable/datadog/templates/autoconf-configmap.yaml

```diff
@@ -3,10 +3,10 @@ kind: ConfigMap
 metadata:
   name: {{ template "datadog.autoconf.fullname" . }}
   labels:
-    app: {{ template "datadog.autoconf.fullname" . }}
-    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
-    release: "{{ .Release.Name }}"
-    heritage: "{{ .Release.Service }}"
+    app: {{ template "datadog.autoconf.fullname" . | quote }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" | quote }}
```

Comment:
Ouch, only the second part will be quoted, not the full field :/

[Other charts](https://github.com/kubernetes/charts/blob/master/stable/cockroachdb/templates/cockroachdb-networkpolicy.yaml#L7-L11) keep quotes for this field

Reply:
d'oh!